core/lib/dal/src/models/proto/mod.proto

```diff
@@ -2,9 +2,23 @@ syntax = "proto3";
 
 package zksync.dal;
 
-import "zksync/roles/validator.proto";
+message Transaction {
+  // Default derive(serde::Serialize) encoding of the zksync_types::Transaction.
+  // TODO(gprusak): it is neither efficient, unique, nor suitable for version control.
+  //     replace with a more robust encoding.
```

Comment:
In this PR? If not, it'd be great to have a task number here, e.g. `TODO(BFT-xxx): ...`.